blocks/rich-text/test/__snapshots__/format.js.snap

```diff
@@ -0,0 +1,23 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`createTinyMCEElement should render a TinyMCE element 1`] = `
+<p
+  a-prop="hi"
+>
+  <p>
```

Comment:
Hm, isn't this invalid HTML? Won't React warn on nesting paragraphs? Or do we not care here for these tests?

Reply:
Our new non-React serializer is much less noisy about issues it encounters. Not to say we shouldn't be using valid HTML here though.